src/containers/home/index.tsx

```diff
@@ -122,40 +128,148 @@ const Home: React.FC = () => {
               </InContain>
             </Col>
           </Row>
+          {privilegeLevel === PrivilegeLevel.ADMIN ? (
```

Comment:
🔧 This doesnt need to be a ternary, we can make it an && instead
```suggestion
          {privilegeLevel === PrivilegeLevel.ADMIN && (
```
And then remove the empty react fragment we return if not an admin

Reply:
Bet